test/usage_test.dart

```diff
@@ -14,6 +14,7 @@ void defineTests() {
     test('simple', () {
       AnalyticsMock mock = new AnalyticsMock();
       mock.sendScreenView('main');
+      mock.sendScreenView('withParameters', parameters: {'cd1': 'custom'});
```

Comment:
I would instead test this in test/usage_impl_test.dart, similar to how setSessionValue is tested.